### PR TITLE
DP-2433: adding exporting documentation

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/content/en/docs/execution/exports/_index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+
+def export_tabular():
+    # Export a particular insight in the desired format (CSV / XLSX)
+    sdk.export.export_tabular_by_insight_id(
+        workspace_id = "demo",
+        insight_id = "revenue",
+        file_format = "CSV",
+        # The flag use_labels ensures that names of columns are according to your model
+        use_labels = True,
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/_index.md:67]: Remove this argument. It is not there anymore.

File: docs/content/en/docs/execution/exports/export_pdf.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: "export_pdf"
+linkTitle: "export_pdf"
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_pdf.md:3]: Do we want to have this title?

File: docs/content/en/docs/execution/exports/export_pdf.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+``export_pdf(workspace_id: str,
+        dashboard_id: str,
+        file_name: str,
+        store_path: Union[str, Path] = Path.cwd(),
+        timeout: float = 60.0,
+        retry: float = 0.2,
+        max_retry: float = 5.0,
+    )``
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_pdf.md:15]: This is not valid code snipet

File: docs/content/en/docs/execution/exports/export_tabular.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+title: "export_tabular"
+linkTitle: "export_tabular"
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_tabular.md:3]: Do we want to have this title?

File: docs/content/en/docs/execution/exports/export_tabular.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+---
+
+``export_tabular(workspace_id: str,
+        dashboard_id: str,
+        file_name: str,
+        store_path: Union[str, Path] = Path.cwd(),
+        timeout: float = 60.0,
+        retry: float = 0.2,
+        max_retry: float = 5.0,
+    )``
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_tabular.md:15]: Not valid code snippet.

File: docs/content/en/docs/execution/exports/export_tabular_by_insight_id.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+{{% parameters-block  title="Parameters" %}}
+{{< parameter p_name="workspace_id" p_type="string" >}}
+The ID of the GoodData Workspace.
+{{< /parameter >}}
+{{< parameter p_name="insight_id" p_type="string" >}}
+The ID of the GoodData Insight.
+{{< /parameter >}}
+{{< parameter p_name="file_name" p_type="String" >}}
+The name of the PDF file (excluding the file extension). Defaults to Path.cwd()
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_tabular_by_insight_id.md:32]: Not true.

File: docs/content/en/docs/execution/exports/export_tabular_by_insight_id.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+{{< /parameter >}}
+{{< parameter p_name="settings" p_type="Optional[ExportSettings]" >}}
+Configuration settings for the export. Defaults to None.
+{{< /parameter >}}
+{{< parameter p_name="store_path" p_type="Union[String, Path]" >}}
+The name of the PDF file (excluding the file extension). Defaults to Path.cwd()
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_tabular_by_insight_id.md:38]: Not true.

File: docs/content/en/docs/execution/exports/export_tabular.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+{{% parameters-block  title="Parameters" %}}
+{{< parameter p_name="workspace_id" p_type="string" >}}
+The ID of the GoodData Workspace.
+{{< /parameter >}}
+{{< parameter p_name="dashboard_id" p_type="string" >}}
+The ID of the GoodData Dashboard.
+{{< /parameter >}}
+{{< parameter p_name="file_name" p_type="String" >}}
+The name of the PDF file (excluding the file extension). Defaults to Path.cwd()
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_tabular.md:28]: Not true.

File: docs/content/en/docs/execution/exports/export_tabular.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+{{< /parameter >}}
+{{< parameter p_name="store_path" p_type="Union[String, Path]" >}}
+The name of the PDF file (excluding the file extension). Defaults to Path.cwd()
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_tabular.md:31]: Not true.

File: docs/content/en/docs/execution/exports/export_pdf.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+{{% parameters-block  title="Parameters" %}}
+{{< parameter p_name="workspace_id" p_type="string" >}}
+The ID of the GoodData Workspace.
+{{< /parameter >}}
+{{< parameter p_name="dashboard_id" p_type="string" >}}
+The ID of the GoodData Dashboard.
+{{< /parameter >}}
+{{< parameter p_name="file_name" p_type="String" >}}
+The name of the PDF file (excluding the file extension). Defaults to Path.cwd()
```
Comment (hkad98) [inline docs/content/en/docs/execution/exports/export_pdf.md:28]: Not true. Does not default to Path.cwd()

File: gooddata-sdk/gooddata_sdk/catalog/export/service.py [hunk 11]
```diff
@@ -21,8 +21,35 @@
...
+            A static method that creates an export and returns its ID.
+        _dashboard_id_exists:
+            Checks if the given dashboard_id exists in the workspace.
+        _export_common:
+            Common export method for handling the exports of PDF and Tabular.
+        export_pdf:
+            Export a PDF of a GoodData Dashboard.
+        export_tabular:
+            Export Tabular data from a GoodData Dashboard.
+    """
```
Comment (hkad98) [inline gooddata-sdk/gooddata_sdk/catalog/export/service.py:44]: Missing **export_tabular_by_insight_id**

Comment (Mara3l) [reply gooddata-sdk/gooddata_sdk/catalog/export/service.py:44]: https://github.com/gooddata/gooddata-python-sdk/pull/298/files/d4ae9dfde8984fe2689b4380eb837a711d4e2e8b#diff-1677d7224ec9ee5fcb82788347dd5058125ffa02c8570f8417b63b86b9dab2fdR277

Comment (hkad98) [reply gooddata-sdk/gooddata_sdk/catalog/export/service.py:44]: I mean the documentation in the context of service.py.

- ExportService docstring

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/catalog/export/service.py (hunk 11 above)
Comment (Mara3l) [reply gooddata-sdk/gooddata_sdk/catalog/export/service.py:44]: 👍

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR298

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/114/ : FAILURE in 1m 15s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/264/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/267/ : SUCCESS in 11s

**Pull request merged**